### PR TITLE
roads to back almost finished

### PR DESCRIPTION
**Pull request by remi59800**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
No dead code

**Review by Dash293 — DISMISSED** [review]

**Review by ThomasDCW — DISMISSED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me

**Review by LoicBrassart — APPROVED** [review]
SnE, good job! ;-)

**Review by ThomasDCW — APPROVED** [review]
Resolve conflicts ... so here we are again

File: backend/src/router.js [hunk 1]
```diff
@@ -6,13 +6,21 @@ const {
...
   UserController,
+  FavoriteController,
   TopicController,
   UserTopicController,
 } = require("./controllers");
 
 const router = express.Router();
 
-//routes /lessons
+
```
Comment (LuigiMenez) [inline backend/src/router.js:16]: useless space

File: backend/src/router.js [hunk 2]
```diff
@@ -6,13 +6,21 @@ const {
...
+router.delete("/users/:id", UserController.delete);
+
+router.get("/favorite/:id", FavoriteController.readByUser);
+router.post("/favorite", FavoriteController.addFavFromUser);
+router.delete("/favorite/:id", FavoriteController.delete);
+
```
Comment (LuigiMenez) [inline backend/src/router.js:22]: Same

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Minor fix

**Review by Dash293 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by LuigiMenez — APPROVED** [review]
ok for me

**Pull request merged**